.registry/resources/minimalaws.ui-schema.yaml

```
@@ -1,4 +1,4 @@
-version: v0.2.0
+version: v0.2.1
 configSections:
 - title: Minimal AWS Configuration
   description: Enter the configuration parameters for your resource pack.
```

I think this should be:

```suggestion
  description: Provide the following details using an existing AWS project.
```

---

I went for `AWS account` since project concept isn't used in AWS AFAIK.